src/controls/Controls.spec.js

```diff
@@ -1 +1,22 @@
 // Test away!
+import React from "react";
+import { render, fireEvent, cleanup } from "react-testing-library";
+import renderer from "react-test-renderer";
+import "react-testing-library/cleanup-after-each";
+import Controls from "./Controls";
+
+describe("<Controls />", () => {
+  it("matches snapshot", () => {
+    const tree = renderer.create(<Controls />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("is mocking me", () => {
+    const mock = jest.fn();
```

Comment:
```suggestion
    const result = jest.fn()();
```
Would this work? Hmm